### ISSUE TITLE
use qtextdocument's MD support instead of hoedown when possible (Qt 5.14+)

### ISSUE THREAD

**Issue by DioEgizio** [PolyMC/PolyMC#963]
Should improve markdown support, especially since hoedown hasn't been updated for almost 10 years and for some reason we're still using it

**Comment by txtsd** [comment]
Took a shot at this. Either got segfaults when opening the About Dialog (the part I was testing), or the markdown was just plaintext with no newlines.

**Comment by DioEgizio** [comment]
oh that's sad.
Also hoedown isn't only used there, also on:
- modrinth modpack browser
- FTB modpack browser
- changelogs on the mod updater
- probably something more I'm forgetting

**Comment by txtsd** [comment]
Yes, I know. I `grep`'d it. The About Dialog was just the first thing I was working on.

Also, the code won't be clean until we bump to 5.15 LTS.

**Comment by flowln** [comment]
Important issue for anyone implementing this in the future:
https://bugreports.qt.io/browse/QTBUG-99366
